src/components/RightPane/CoursePane/CourseRenderPane.tsx

```diff
@@ -277,27 +273,30 @@ class CourseRenderPane extends PureComponent<CourseRenderPaneProps, CourseRender
             };
 
             currentView = (
-                <div className={classes.root}>
+                <>
                     <RecruitmentBanner {...classes} />
-                    {this.state.courseData.length === 0 ? (
-                        <div className={classes.noResultsDiv}>
-                            <img src={isDarkMode() ? darkNoNothing : noNothing} alt="No Results Found" />
-                        </div>
-                    ) : (
-                        this.state.courseData.map((_: School | Department | AACourse, index: number) => {
-                            let heightEstimate = 200;
-                            if ((this.state.courseData[index] as AACourse).sections !== undefined)
-                                heightEstimate =
-                                    (this.state.courseData[index] as AACourse).sections.length * 60 + 20 + 40;
+                    <div className={classes.root} style={{ position: 'relative' }}>
+                        <div className={classes.spacing} />
```

Comment:
This div is literally just there to space the back and reload buttons so they don't overlap.
<img width="109" alt="image" src="https://user-images.githubusercontent.com/53128285/231310995-ee2b6d3c-f7d3-4d05-904c-26c623bd16f6.png">
<img width="228" alt="image" src="https://user-images.githubusercontent.com/53128285/231311042-92e909ea-b191-4c70-a3e8-afed4b7b9002.png">

If anyone else knows a better way to do this LMK.

Reply:
🤷‍♂️